### PR TITLE
LPS-199793 Add POSHI automation | Master

### PR DESCRIPTION
**Pull request by alessandromalves**
This is only for master
https://liferay.atlassian.net/browse/LPS-199793 

cc.: @david-gutierrez-mesa @Tim-Cao

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/pagetemplate/masterspagetemplate/MastersPageTemplates.testcase [hunk 1]
```diff
@@ -1943,6 +1943,82 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-199793. Using Keyboard navigation it is possible to copy an existing Master Page from the ellipsis menu of the card."
+	@priority = 4
+	test viewMasterPagerCanBeCopiedUsingKeyboardNavigation {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/pagetemplate/masterspagetemplate/MastersPageTemplates.testcase:1948]: Move the case to PageTemplatesAdminWithKeyboard.testcase then rename to CopyMasterPageTemplateViaKeyboard

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/pagetemplate/masterspagetemplate/MastersPageTemplates.testcase [hunk 2]
```diff
@@ -1943,6 +1943,82 @@ definition {
...
+		task ("Given a master page") {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/pagetemplate/masterspagetemplate/MastersPageTemplates.testcase:1949]: Given a user has a master page

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/pagetemplate/masterspagetemplate/MastersPageTemplates.testcase [hunk 3]
```diff
@@ -1943,6 +1943,82 @@ definition {
...
+			MastersPageTemplatesAdmin.gotoMasters(masterLayoutName = "Custom Masters Page Template");
+
+			MastersPageTemplates.publishMaster();
+		}
+
+		task ("When the user uses the keyboard keys to navigate to the ellipsis menu options") {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/pagetemplate/masterspagetemplate/MastersPageTemplates.testcase:1955]: When the user accesses to the Masters admin

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/pagetemplate/masterspagetemplate/MastersPageTemplates.testcase [hunk 4]
```diff
@@ -1943,6 +1943,82 @@ definition {
...
+
+			for (var menuItem : list "Edit,Change Thumbnail,Mark as Default,Rename,Export") {
+				KeyPress(
+					key_menuItem = ${menuItem},
+					locator1 = "MenuItem#ANY_MENU_ITEM",
+					value1 = "\DOWN");
+			}
+		}
+
+		task ("Then the user will be able to make a copy of the created master page") {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/pagetemplate/masterspagetemplate/MastersPageTemplates.testcase:2010]: Then the user could copy master page via keyboard

**Review by Tim-Cao — CHANGES_REQUESTED** [review]
Hi @alessandromalves, would you mind modifying according to comments? Thanks

**Comment by alessandromalves** [comment]
ci:test:sf

**Comment by alessandromalves** [comment]
ci:test:sf

**Comment by Tim-Cao** [comment]
@alessandromalves See changes https://github.com/liferay-echo/liferay-portal/pull/14156

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**